app/views/support/service_performance/_mno_rollout.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="govuk-heading-l">MNO rollout</h2>
```

Comment:
Is `MNO rollout` how most people refer to it? Just want to make sure this is what makes sense to most people. I've heard different variations of MNO rollout, MNO offer and Connectivity offer now.